### PR TITLE
Rewrite test templates

### LINKED ISSUE #90: Rewrite test templates
Rewrite test templates so that they are automatically generated.

### PR DESCRIPTION
**Pull request by IraSoro**
Closed #90 

New templates have been added. With them, you can view more options: different phases, days of ovulation and menstruation. And also in these functions there is an argument - a number, which means how many cycles to generate.

### CODE REVIEW

File: src/data/Storage.ts [hunk 1]
```diff
@@ -54,133 +54,147 @@ export const storage = {
...
+// _delayOfCycle(6).catch((err) => console.error(err));
 
 function _emptyArrayOfCycles(): Promise<void> {
   return storageImpl.remove("cycles") as Promise<void>;
 }
 
-function _arrayOfCyclesWithSingleCycle(): Promise<void> {
-  return storage.set.cycles([
-    {
+function _todayPeriod(countOfCycles: number): Promise<void> {
```
Comment (imblowfish) [inline src/data/Storage.ts:70]: Do we really needed in `countOfCycles` for a simple template functions which we use just for functionality check?

File: src/data/Storage.ts [hunk 2]
```diff
@@ -54,133 +54,147 @@ export const storage = {
...
+  for (let i = 0; i < countOfCycles; ++i) {
+    date.setDate(date.getDate() - 28);
+    cycles.push({
       cycleLength: 28,
       periodLength: 6,
-      startDate: "2023-06-30",
-    },
-  ] satisfies Cycle[]);
+      startDate: date.toString(),
+    });
```
Comment (imblowfish) [inline src/data/Storage.ts:82]: You remove but forgot to return `satisfies` here

File: src/data/Storage.ts [hunk 3]
```diff
@@ -54,133 +54,147 @@ export const storage = {
 
 // NOTE: Predefined templates for test purpose
 //       for use just uncomment one of the following lines:
-// _emptyArrayOfCycles().catch((err) => console.error(err));
-// _arrayOfCyclesWithSingleCycle().catch((err) => console.error(err));
-// _arrayOfCyclesWithSingleCycleWithDelay().catch((err) => console.error(err));
-// _halfFilledArrayOfCycles().catch((err) => console.error(err));
-// _fullyFilledArrayOfCycles().catch((err) => console.error(err));
-// _fullyFilledArrayOfCyclesWithDelay().catch((err) => console.error(err));
+
```
Comment (imblowfish) [inline src/data/Storage.ts:57]: Why did you remove `emptyArray` template? If you decided to replace this function with `countOfCycles` parameter then I dont agree with that. Previously we had just one function which makes cycles array empty, but now we have multiple function with the same parameter which also can make array empty.

**Review by imblowfish — CHANGES_REQUESTED** [review]

File: src/data/Storage.ts (hunk 3 above)
Comment (IraSoro) [reply src/data/Storage.ts:57]: This function hasn't been removed. I forgot to leave her challenge in the comments. 
Fixed.

File: src/data/Storage.ts (hunk 2 above)
Comment (IraSoro) [reply src/data/Storage.ts:82]: In my situation it isn`t needed. I removed this.

File: src/data/Storage.ts (hunk 1 above)
Comment (IraSoro) [reply src/data/Storage.ts:70]: This is an important parameter. So we can see on the "Details" what a different number of cycles will look like.

Comment (imblowfish) [reply src/data/Storage.ts:71]: Yes, but we already have a function which generates empty array. Maybe it's better to modify this function and let other functions just create cycles which we waiting from them?
For example, from ovulationToday function we waiting for situation when ovulation is today and it's doesnt matter how much cycles it generates. Same for another functions. But if we introduce function which generates required number of cycles that will be more clear in my opinion.

**Review by imblowfish — APPROVED** [review]
